src/VisualStudio/Core/Def/Progression/GraphQueries/InheritedByGraphQuery.cs

```diff
@@ -41,7 +41,7 @@ public async Task<GraphBuilder> GetGraphAsync(Solution solution, IGraphContext c
                     namedType, solution, transitive: false, cancellationToken: cancellationToken).ConfigureAwait(false);
                 var derivedInterfaces = await SymbolFinder.FindDerivedInterfacesArrayAsync(
                     namedType, solution, transitive: false, cancellationToken: cancellationToken).ConfigureAwait(false);
-                foreach (var derivedType in implementingClassesAndStructs.Concat(derivedInterfaces))
+                foreach (var derivedType in implementingClassesAndStructs.ConcatFast(derivedInterfaces))
```

Comment:
>ConcatFast [](http://example.com/codeflow?start=74&length=10)

Would it be better to just call the non array versions of the methods on SymbolFinder instead?

Reply:
No. The non array versions just call the array versions and then box them. :-)

Reply:
OK, I see. It's a public contract that's why the enumerable version exists.